src/server.js

```diff
@@ -55,15 +42,10 @@ app.get('/export', (req, res) => {
 })
 
 app.get('/ls', (req, res) => {
-    debug = !debug && checkDebug(req,res);
-
     var longList = ""
     if (req.query.ll)
         longList = "-l";
 
-// address is optional don't know how to test
```

Comment:
Just real quick on this one. I wanted to allow the user to specify an address here, but it's optional, so I didn't know how to test for optionality here. Can you put this back in, but allow for both empty and 42 characters?

Reply:
It belongs as an issue. I'll invite you to create the issue.